models.py

```diff
@@ -305,10 +306,23 @@ class Erfolg(Verknuepfung):
         on_delete=models.PROTECT,
         null=True,
     )
+    zusatz = models.CharField(
+        max_length=99,
+        null=True,
```

Comment:
CharField braucht i.A. kein null, es sei denn es gäbe einen triftigen Grund zwischen die beiden Zustände "" und None, die beide „leer“ bedeuten, zu unterscheiden; gibt's den?

Reply:
in der Anzeige sollte ja kein Komma angezeigt werden wenn es keinen Zusatz gibt

Reply:
das ist klar, aber der leere String tut's auch. Deine Prüfung ´if self.zusatz´ würde bei "" und bei None das gleiche Ergebnis geben. Und es gibt keinen Grund, zwei verschiedene Arten, nichts darzustellen, zu haben - oder? :)